### PR TITLE
Player proto

### PR DESCRIPTION
**Pull request by DIAMONDUSNITRO**
image

### CODE REVIEW

**Comment by haennes** [comment]
Did you change anything on this file.
It seems like you just copied it from:
https://sketchfab.com/3d-models/male-base-model-c613f570ecda43628018e78ef932b89d
I don't think that's a problem.
The licence is actually not that good as seen here:
https://creativecommons.org/licenses/by/4.0/

File: Mchar.gltf [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        0
+      ],
+      "type": "SCALAR"
+    }
+  ],
+  "asset": {
+    "extras": {
+      "author": "gome70 (https://sketchfab.com/gome70)",
+      "license": "CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)",
+      "source": "https://sketchfab.com/3d-models/male-base-model-c613f570ecda43628018e78ef932b89d",
```
Comment (haennes) [inline Mchar.gltf:85]: This is the author, license, source of the model

Comment (DIAMONDUSNITRO) [reply Mchar.gltf:85]: pretty much yes, it's a screenshot 
just if this style would be convenient or something in lower poly

**Comment by DIAMONDUSNITRO** [comment]
pretty much a copy/screenshot if this style is convenient or not

**Pull request merged**